fiaas_deploy_daemon/deployer/kubernetes/deployment/deployer.py

```diff
@@ -85,9 +85,9 @@ def deploy(self, app_spec, selector, labels, besteffort_qos_is_required):
             try:
                 deployment = Deployment.get(app_spec.name, app_spec.namespace)
                 replicas = deployment.spec.replicas
+                LOG.warn("configured replica size (%d) for deployment is being ignored, as current running replica size"
+                         " is different (%d) for %s", deployment.spec.replicas, app_spec.replicas, app_spec.name)
```

Comment:
The order of the parameters doesn't match the text if I read it correctly.

Also, can we start the sentence with a capital letter please?

I'm also wondering if it should be a warning, or just info?

Most of the time, I think we are ignoring the config because the autoscaler has scaled the application up. It is only a surprise/problem if the current value is 0 and the user expected it to start running. Actually the general problem is if the current number of replicas is lower than your defined minimum, that is something that should be warned about. Otherwise it's just useful information.

Reply:
Feedback now incorporated to PR. Let me know if i should squash the commits before you want to merge, or feel free to just do it yourself. Also if there is anything else missing.